src/components/Shell/Shell.stories.js

```diff
@@ -29,7 +29,7 @@ const shellProps = {
   toolbar,
 };
 
-disableCentered(storiesOf(patterns('Shell'), module))
+disableCentered(storiesOf(patterns('Shell (Security version)'), module))
```

Comment:
This is minor, but what are your thoughts around just referring to this as `Shell (Security)` and the other as `UIShell (Carbon)`?